spec/unit/recipes/_configure_server_spec.rb

```diff
@@ -0,0 +1,354 @@
+    let(:expected_active_group) { 'Trial' }
+    let(:expected_clustering) do
+      {
+        'clustering' => {
+          'mode' => 'searchhead',
+          'master_uri' => 'clustermaster:cerner_splunk/singlesite_cluster,clustermaster:cerner_splunk/multisite_cluster'
+        },
+        'clustermaster:cerner_splunk/multisite_cluster' => {
+          'master_uri' => 'https://cluster-master:8089',
+          'multisite' => true,
+          'site' => 'site1'
+        },
+        'clustermaster:cerner_splunk/singlesite_cluster' => {
+          'master_uri' => 'https://cluster-master:8089'
+        }
+      }
+    end
+
+    it 'writes the server.conf file with the appropriate configs' do
+      expect(subject).to create_splunk_template('system/server.conf').with(stanzas: expected_attributes)
+      expect(subject.splunk_template('system/server.conf')).to notify('file[splunk-marker]').to(:touch)
+    end
+  end
+
+  context 'when node is a search head' do
+    let(:node_type) { :search_head }
+    let(:expected_active_group) { 'Trial' }
+    let(:expected_clustering) do
+      {
+        'clustering' => {
+          'master_uri' => 'clustermaster:cerner_splunk/multisite_cluster,clustermaster:cerner_splunk/singlesite_cluster',
+          'mode' => 'searchhead'
+        },
+        'clustermaster:cerner_splunk/multisite_cluster' => {
+          'master_uri' => 'https://cluster-master:8089',
+          'multisite' => true,
+          'site' => 'site1'
+        },
+        'clustermaster:cerner_splunk/singlesite_cluster' => {
+          'master_uri' => 'https://cluster-master:8089'
+        }
+      }
+    end
+
+    it 'writes the server.conf file with the appropriate configs' do
+      expect(subject).to create_splunk_template('system/server.conf').with(stanzas: expected_attributes)
+      expect(subject.splunk_template('system/server.conf')).to notify('file[splunk-marker]').to(:touch)
+    end
+  end
+
+  context 'when node is a server' do
+    let(:node_type) { :server }
+    let(:expected_active_group) { 'Trial' }
+    let(:expected_clustering) do
+      {
+        'clustering' => {
+          'master_uri' => 'clustermaster:cerner_splunk/multisite_cluster,clustermaster:cerner_splunk/singlesite_cluster',
+          'mode' => 'searchhead'
+        },
+        'clustermaster:cerner_splunk/multisite_cluster' => {
+          'master_uri' => 'https://cluster-master:8089',
+          'multisite' => true,
+          'site' => 'site1'
+        },
+        'clustermaster:cerner_splunk/singlesite_cluster' => {
+          'master_uri' => 'https://cluster-master:8089'
+        }
+      }
+    end
+
+    it 'writes the server.conf file with the appropriate configs' do
+      expect(subject).to create_splunk_template('system/server.conf').with(stanzas: expected_attributes)
+      expect(subject.splunk_template('system/server.conf')).to notify('file[splunk-marker]').to(:touch)
+    end
+  end
+
+  context 'when node is a shc_captain' do
```

Comment:
This seems nearly the same as the next context on line 322. If they are different then the context description needs to call out the difference, or use a sub-context.

Reply:
I think that was a copy/paste error which created duplicate test. Removed it in 3819e106afcd180039e210ceb0cc8cc764a2ba7a